docs/deployment/k8s/index.mdx

```diff
@@ -60,29 +64,49 @@ config:
 helm upgrade --install -f values.yaml nats-cluster nats/nats
 ```
 
-#### Running wadm
+#### Deploying wadm
 
-You can deploy wadm to your Kubernetes cluster with the wadm Helm chart. First, clone the wadm repo from GitHub:
+You can deploy wadm to your Kubernetes cluster with a Helm chart hosted by wasmCloud. Use this `values.yaml` file to define a minimal deployment that connects with the NATS server deployed above: 
 
-```shell
-git clone https://github.com/wasmCloud/wadm.git
+```yaml
+wadm:
+  config:
+    nats:
+      server: "nats-cluster.default.svc.cluster.local:4222"
 ```
 
-In the cloned repo, find the chart at `wadm/charts/wadm`. In your `values.yaml` file, the `server` field for NATS should be set to:
+Run `helm install`:
 
-```yaml
-server: 'nats-cluster.default.svc.cluster.local:4222'
+```shell
+helm install wadm -f values.yaml --version 0.2.2 oci://ghcr.io/wasmcloud/charts/wadm
 ```
 
-Run `helm install` from `wadm/charts/wadm` in the cloned repo:
+#### Start the operator
+
+To download and deploy the operator:
 
 ```shell
-helm install wadm -f values.yaml ./
+helm install wasmcloud-operator --version 0.1.3 oci://ghcr.io/wasmcloud/charts/wasmcloud-operator

```

Comment:
Same comment as above, as I'm sure we'll keep improving the wasmcloud-operator chart
```suggestion
helm install wasmcloud-operator oci://ghcr.io/wasmcloud/charts/wasmcloud-operator
```